cpp/src/io/copc_writer_internal.cpp

```diff
@@ -60,10 +63,10 @@ void WriterInternal::Close()
     WritePageTree(hierarchy_->seen_pages_[VoxelKey::RootKey()]);
 
     // If WKT is provided, write it as EVLR
-    if (!copc_config_->Wkt().empty())
+    if (!config_->Wkt().empty())
```

Comment:
wkt should be written in the base class?

Reply:
or refactor into a base class function `WriteWKT()`

Reply:
Refactored in a base class.